client/app/controllers/login-error.js

```diff
@@ -0,0 +1,11 @@
+import Controller from '@ember/controller';
+
+export default class LoginErrorController extends Controller {
+  /**
+    * @return      {bool} true if login route yields error response
+    *                     about Contact not existing in CRM.
+    */
+  get contactNotAssigned() {
+    return this.model.errors.some((error) => error.response.code === 'NO_CONTACT_FOUND');
```

Comment:
Is `NO_CONTACT_FOUND` the error code CRM sends to our API, or are we defining "NO_CONTACT_FOUND" as the error code that our API creates?

Reply:
The latter -- an error code that our API creates!

Reply:
Nevermind!
> And the frontend can then use the error.response.code property (APPLICANT_PORTAL_SPECIFIC_CODE) to conditionally display more user friendly messages.